### PR TITLE
Complete sign and submit 187003764

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

**Comment by bseeger** [comment]
Related to this PR.  [This line's fonts](https://github.com/codeforamerica/mdpilot/blob/a3789042d48c1860e59aaafee61326b61b076db0/src/main/resources/templates/mdBenefitsFlow/legalStuff.html#L16) are smaller than the design.  I'm wondering if you can try it as a subheader in the cardHeader font: 

```
cardHeader(header=#{legal-stuff.header}, subheader=#{legal-stuff.help})
```

**Comment by bseeger** [comment]
Related to this PR: 
The wording on the http://localhost:8080/flow/mdBenefitsFlow/sensitiveConvictionQuestionsIntro is different from the design.  The apps page has more text than the design has.

**Comment by bseeger** [comment]
Also related to this PR: 
In the "Your Right's" section on the rightsAndResponsibility page all the page links to external references are missing. 

Under "TCA and SNAP Penalties" - the first line ("Do not:") should be bold.

**Comment by bseeger** [comment]
the `rightsAndResponsibilites` page is missing the "Medicaid Warning and Penalties" section.

**Comment by bseeger** [comment]
The first, second and signature page seems different from the design. Not sure what's changed.  I know I listed things outside of this PR, but I thought this PR was completing this section and it looks like there is more work to be done.  Separate ticket?

**Review by bseeger — COMMENTED** [review]
I have one more comment about what agreement I should see when. If I choose TDAP only, I get an OHEP agreement (only). Wasn't sure if that was supposed to happen?

**Review by bseeger — APPROVED** [review]
Minor comments. Feel free to merge once addressed.

File: src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java [hunk 1]
```diff
@@ -16,6 +16,6 @@ public class SetDefaultSensitiveConvictionAnswers implements Action {
 
     @Override
     public void run(Submission submission) {
-        Arrays.stream(SubmissionUtilities.SENSITIVE_CONVICTION_QUESTIONS).forEach(s -> submission.getInputData().putIfAbsent("noOne" + s + "[]", "true"));
+        Arrays.stream(SubmissionUtilities.SENSITIVE_CONVICTION_QUESTIONS).forEach(s -> submission.getInputData().putIfAbsent("someone" + s, "false"));
```
Comment (analoo) [inline src/main/java/org/mdbenefits/app/submission/actions/SetDefaultSensitiveConvictionAnswers.java:19]: I love this pattern.

File: src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java [hunk 2]
```diff
@@ -394,10 +394,17 @@ void docUploadSkipTest() {
     @Test
     void sensitiveConvictionQuestionsArePreCheckedTest() {
```
Comment (analoo) [inline src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java:395]: Small thing:

sensitiveConvictionQuestionsArePrechecked**No**Test

**Review by analoo — APPROVED** [review]

**Comment by analoo** [comment]
PR looked good to me. Only general feedback is a question around web addresses: Do we always link the actual page or is that something we only do if the design requires it. For example there are a few places in rights and responsibilities where I can see a URL but I don't have the ability to select it:

Screenshot 2024-02-15 at 1 55 02 PM

File: src/main/resources/templates/mdBenefitsFlow/sensitiveConvictionQuestions.html [hunk 3]
```diff
@@ -15,15 +15,20 @@
           <th:block th:ref="sensitiveQuestionsContent">
             <div class="form-card__content"
                  th:each="question : ${T(org.mdbenefits.app.utils.SubmissionUtilities).SENSITIVE_CONVICTION_QUESTIONS}">
-              <div th:with="inputName='noOne' + ${question}, label='sensitive-conviction-question-' + ${question}">
-                <th:block th:replace="~{fragments/inputs/checkboxFieldset ::
-                  checkboxFieldset(inputName=${inputName},
-                  label=#{${label}},
-                  content=~{::checkboxInSetContent})}">
-                  <th:block th:ref="checkboxInSetContent">
+              <div th:with="inputName='someone' + ${question}, label='sensitive-conviction-question-' + ${question}, helpText='sensitive-conviction-question-' + ${question} + '-helpText'">
```
[the remark this answers is not in the document]
Comment (vrajmohan) [reply src/main/resources/templates/mdBenefitsFlow/sensitiveConvictionQuestions.html:18]: Good catch. Updating.

File: src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java (hunk 2 above)
Comment (vrajmohan) [reply src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java:395]: Good catch, went with `sensitiveConvictionQuestionsArePreCheckedToFalseTest`

**Comment by vrajmohan** [comment]
@analoo , I missed a spot in formatting URLs.

**Pull request merged**